content/android/get-started.md

```diff
@@ -0,0 +1,202 @@
+---
+title: "Get Started"
+date: 2017-07-03T22:20:00-04:00
+draft: false
+mainpage: false
+projectpage: true
+projectslug: "android"
+projectname: "WorldWind Android"
+listdescription: ""
+layout: "project"
+---
+
+## Getting Started
+
+WorldWind Android is an [Android Library](https://developer.android.com/studio/projects/android-library.html) which compiles into an Android Archive (AAR) file. The AAR can be linked and used to display the 3D virtual globe within your application. The WorldWind Android AAR is compatible to API level 19 (Android 4.4 KitKat).
+
+Skip to the [**Download**](#download) section for directions on how to add WorldWind Android to your application. If you need some guidance with setting up the Android Studio IDE, the [**Setup**](#setup) section includes in-depth directions and walks through the creation of a sample project utilizing the WorldWind library.
```

Comment:
I think it's best to omit this description and let the headings to the talking. People don't really read web pages, they scan them. Extra text just makes things look more complex than they really are.